framework/src/main/java/org/tron/core/config/args/Args.java

```diff
@@ -985,17 +985,9 @@ public static void logConfig() {
   }
 
   public static void initConfig(CommonParameter parameter) {
-    initVMConfig(parameter);
     initDBConfig(parameter);
   }
 
-  public static void initVMConfig(CommonParameter parameter) {
-    VMConfig.setMaxTimeRatio(parameter.getMaxTimeRatio());
```

Comment:
where is this logic in the new implementation?

Reply:
maxTimeRatio in vm config already delate, directly using the variable in common parameter.

Reply:
ok, I see